var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -124,8 +124,8 @@ class Vtk(CMakePackage):
     patch("vtk_movie_link_ogg.patch", when="@8.2")
     patch("vtk_use_sqlite_name_vtk_expects.patch", when="@8.2")
     patch("vtk_proj_include_no_strict.patch", when="@9: platform=windows")
-    patch("vtk_alias_hdf5.patch", when="@9: platform=windows")
-    patch("vtk_findproj_config.patch", when="platform=windows")
+    patch("vtk_findproj_config.patch")
```

Comment:
This line had a conflict on rebase so I made a change here.
@Chrismarsh we had discussed the need for this patch to run on all platforms, not just Windows right?
If not I'll revert this change.

Reply:
It looks like this patch is not needed from my testing so far. I also see it fails the vtk@8 CI